onboarding/src/Components/EmployeeProfile/FormsSentTableRow.js

```diff
@@ -1,8 +1,8 @@
 import React, { useState } from "react";
 
 function FormsSentTableRow(props) {
-    const [toggleObj, switchVisibility] = useState({text: String.fromCharCode(8631), style:{ display: "none" }, display: false, hasContent: props.empty? false : props.row.pagesCount > 0});
-
+    const [toggleObj, switchVisibility] = useState({style:{ display: "none" }, display: false, hasContent: props.empty? false : props.row.pagesCount > 0});
+    const [rotate, setCaretIcon] = useState(false);
```

Comment:
What would be wrong in putting this value into `toggleObj` object as a property(?) :wink: 
(Don't feel to be pushed. It's just a suggestion :) )